lib/components/artist/biography.js

```diff
@@ -44,6 +44,19 @@ class Biography extends React.Component {
   }
 }
 
+const styles = StyleSheet.create({
+  blurb: {
+    fontSize: 16, 
+    lineHeight: 20,
+    marginBottom: 20
+  }, 
+  bio: {
+    fontSize: 16, 
+    lineHeight: 20,
+    marginBottom: 40
+  }
+});
```

Comment:
FYI (not a blocker) the common ones could have gone in a common style and then you’d pass an array of styles to the `style` prop, RN will then merge those. E.g.

```
const styles = StyleSheet.create({
  text: {
    fontSize: 16,
    lineHeight: 20,
  },
  blurb: {
    marginBottom: 20,
  }
});

<SerifText style={[styles.text, styles.blurb]} numberOfLines={0}>...</SerifText>
```